packages/qgrid-ngx-examples/src/examples/focus-cell-custom/example-focus-cell-custom.component.scss

```diff
@@ -0,0 +1,7 @@
+.margin {
```

Comment:
why we need those margin and padding, and short way:
```css
.margin {
   margin: 5px;
}
```

Reply:
This margins & padding so it looks better in the demo.